src/libcore/object.cpp

```diff
@@ -38,5 +38,7 @@ std::ostream& operator<<(std::ostream &os, const Object *object) {
     return os;
 }
 
+void Object::set_grad_suspended(bool state) { }
```

Comment:
put `state` parameter name in comment: `/* state */`, otherwise you'll get a unused parameter warning on some platforms.